server/types/schemas.ts

```diff
@@ -1,10 +1,11 @@
-import { ObjectId } from '../deps.ts';
+import { ObjectId } from "../deps.ts";
 
 export interface RoomSchema {
   _id: ObjectId;
   roomCode: string;
   moderatorId: ObjectId;
   options: number[];
+  voters: string[];
```

Comment:
Since we're adding this as a required property for a Room, I think we'll need to add it to the Room create route as part of the rooms.insertOne()